deploy/docker/seqr/Dockerfile

```diff
@@ -80,7 +80,13 @@ RUN echo "deb http://packages.cloud.google.com/apt cloud-sdk-bullseye main" > /e
 COPY --from=build /opt/venv /opt/venv
 
 WORKDIR /seqr
-COPY . .
+COPY admin/ .
+COPY matchmaker/ .
+COPY panelapp/ .
+COPY reference_data/ .
+COPY seqr/ .
+COPY static/ .
+COPY ui/ .
```

Comment:
actually, I'm not sure we need ui here because we copy the build ui below, but I would double check that